core/config/project_settings.cpp

```diff
@@ -1570,6 +1570,8 @@ ProjectSettings::ProjectSettings() {
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "rendering/textures/canvas_textures/default_texture_filter", PROPERTY_HINT_ENUM, "Nearest,Linear,Linear Mipmap,Nearest Mipmap"), 1);
 	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "rendering/textures/canvas_textures/default_texture_repeat", PROPERTY_HINT_ENUM, "Disable,Enable,Mirror"), 0);
 
+	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "rendering/viewport/default_canvas_cull_mask", PROPERTY_HINT_LAYERS_2D_RENDER), 4294967295L);
```

Comment:
```suggestion
	GLOBAL_DEF_BASIC(PropertyInfo(Variant::INT, "rendering/viewport/default_canvas_cull_mask", PROPERTY_HINT_LAYERS_2D_RENDER), 4294967295);
```
Maybe this one fails CI? Idk actually.

Reply:
Without L was the first Variant, and it still fails